packages/report-server/src/reportBuilder/transform/functions/excludeColumns.ts

```diff
@@ -20,8 +21,8 @@ const paramsValidator = yup.object().shape({
   where: yup.string(),
 });
 
-const excludeColumns = (rows: Row[], params: ExcludeColumnsParams): Row[] => {
-  const parser = new TransformParser(rows);
+const excludeColumns = (rows: Row[], params: ExcludeColumnsParams, context: Context): Row[] => {
```

Comment:
I agree that while there are solutions to inject `context` in those functions, they will probably make the code more complicated, as it will be harder to follow where the context is coming from... And you have established a nice functional pattern here!

There is a tipping point where the complexity of a wrapper would be preferred over the complexity/clutter of an additional parameter like context, but I believe we haven't reached that stage yet 🙂